public/javascripts/SVValidate/src/panorama/GSVOverlay.js

```diff
@@ -0,0 +1,143 @@
+        panningDisabled = false;
+    }
+
+    /**
+     * This is a callback function that is fired with the mouse down event
+     * on the view control layer (where you control street view angle.)
+     * @param e
+     */
+    function handlerViewControlLayerMouseDown (e) {
+        mouseStatus.isLeftDown = true;
+        mouseStatus.leftDownX = mouseposition(e, this).x;
+        mouseStatus.leftDownY = mouseposition(e, this).y;
+        viewControlLayer.css("cursor", "url(/assets/javascripts/SVLabel/img/cursors/closedhand.cur) 4 4, move");
+
+        //This is necessary for supporting touch devices, because there is no mouse hover
+        mouseStatus.prevX = mouseposition(e, this).x;
+        mouseStatus.prevY = mouseposition(e, this).y;
+    }
+
+    /**
+     * This is a callback function that is called with mouse up event on
+     * the view control layer (where you change the Google Street view angle.
+     * @param e
+     */
+    function handlerViewControlLayerMouseUp (e) {
+        viewControlLayer.css("cursor", "url(/assets/javascripts/SVLabel/img/cursors/openhand.cur) 4 4, move");
+        mouseStatus.isLeftDown = false;
+        mouseStatus.leftUpX = mouseposition(e, this).x;
+        mouseStatus.leftUpY = mouseposition(e, this).y;
+
+    }
+    /**
+     * Handles mouse leaving control view.
+     * @param e
+     */
+    function handlerViewControlLayerMouseLeave (e) {
+        viewControlLayer.css("cursor", "url(/assets/javascripts/SVLabel/img/cursors/openhand.cur) 4 4, move");
+        mouseStatus.isLeftDown = false;
+    }
+
+    /**
+     * This is a callback function that is fired when a user moves a mouse on the
+     * view control layer where you change the pov.
+     */
+    function handlerViewControlLayerMouseMove (e) {
+        mouseStatus.currX = mouseposition(e, this).x;
+        mouseStatus.currY = mouseposition(e, this).y;
+
+        var timestamp = new Date().getTime();  // waits till the pano is fully loaded
+        if ((timestamp - svv.panorama.getProperty("prevSetPanoTimestamp") > 800) && mouseStatus.isLeftDown && panningDisabled === false) {
+            // If a mouse is being dragged on the control layer, move the sv image.
+            var dx = mouseStatus.currX - mouseStatus.prevX;
+            var dy = mouseStatus.currY - mouseStatus.prevY;
+            var pov = svv.panorama.getPov();
+            var zoomLevel = pov.zoom;
+            dx = dx / (2 * zoomLevel);
+            dy = dy / (2 * zoomLevel);
+            dx *= 1.5;
+            dy *= 1.5;
+            updatePov(dx, dy);
+        }
+        mouseStatus.prevX = mouseposition(e, this).x;
+        mouseStatus.prevY = mouseposition(e, this).y;
+    }
+
+    /**
+     * Update POV of Street View as a user drag a mouse cursor.
+     * @param dx
+     * @param dy
+     */
+    function updatePov (dx, dy) {
+        var pano = svv.panorama.getPanorama();
+        if (pano) {
+            var pov = pano.getPov(),
+                alpha = 0.25;
```

Comment:
Instead of
```
var pov = pano.getPov(),
    alpha = 0.25;
```
can you doi
```
var pov = pano.getPov(),
var alpha = 0.25;
```

But also use `let` instead lol